brands/shop/convenience.json

```diff
@@ -1765,6 +1765,27 @@
       "name": "Дружба",
       "shop": "convenience"
     }
+  },
+   "shop/convenience|Евроопт Market": {
+    "countryCodes": ["by"],
+    "matchNames": ["Евроопт Маркет"],
+     "tags": {
+      "brand": "Евроопт Market",
+      "brand:wikidata": "Q65455869",
+      "brand:wikipedia": "be:Eurotorg",
+      "name": "Евроопт Market",
```

Comment:
It’s branded as Market, but since pronunciation of both en and ru/by variants would be the same, I though to match any people could put in using Cyrillic. I don’t think there are any in the database with the Cyrillic subbrand spelling. 
Thanks for pointing out the wiki, will remove it